sdk/openai/openai/tests.yml

```diff
@@ -5,6 +5,10 @@ extends:
   parameters:
     PackageName: "@azure/openai"
     ServiceDirectory: openai
+    Clouds: 'Public'
```

Comment:
Clouds: Public is the default so isn't needed.

Reply:
Thanks, I opened https://github.com/Azure/azure-sdk-for-js/pull/31415